src/main/java/bookshop/controller/BookController.java

```diff
@@ -0,0 +1,37 @@
+package bookshop.controller;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import bookshop.dto.BookDto;
+import bookshop.dto.CreateBookRequestDto;
+import bookshop.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
+@RequiredArgsConstructor
+public class BookController {
+    private final BookService bookService;
+
+    @GetMapping(produces = APPLICATION_JSON_VALUE)

```

Comment:
JSON is a default response type, so it's not necessary to specify it explicitly.
```suggestion
    @GetMapping
```